includes/admin/class-wcmp-export.php

```diff
@@ -823,55 +826,121 @@ public function saveShipmentData(WC_Order $order, array $shipment): void
     }
 
     /**
-     * TODO: For MyParcel NL, currently not necessary for BE.
+     * @param array  $order
+     * @param string $shipping_method_id
      *
+     * @return bool|int
```

Comment:
als je niet een int kan teruggeven moet je een null teruggeven. Dan is je returntype int|null. en achter op regel 835 ?int

Reply:
Done